libraries/core/src/test/java/com/paritytrading/philadelphia/FIXInitiatorPipeTest.java

```diff
@@ -0,0 +1,24 @@
+package com.paritytrading.philadelphia;
+
+import java.io.IOException;
+import java.nio.channels.Pipe;
+import org.jetbrains.annotations.NotNull;
```

Comment:
Let's drop this annotation, it's not been used in this project before.

Reply:
Apologies, when IDEA sees it in the classpath, it automatically adds it :-)